### PR TITLE
Plain development

### PR DESCRIPTION
**Pull request by initseis**
In this branch we:

- Created a simple HTML UI
- Created an Object Array
- Added event listeners to display books when are added and removed
- Created Local Storage

We are having problems with the Webhint linter, It is showing us errors in lines that we do not have.
imagen

### CODE REVIEW

File: scripts/main.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  removeButton.className = 'remove-button';
+  removeButton.innerHTML = 'Remove';
+  removeButton.id = localStorageBooks[i].id;
+  bookLi.appendChild(removeButton);
+  const hrLi = document.createElement('hr');
+  bookLi.appendChild(hrLi);
+  removeButton.addEventListener('click', removeBook);
+}
+
+function addBook() {
```
Comment (MahmoudBakr23) [inline scripts/main.js:59]: - [ ] **OPTIONAL**
When adding an empty book, it succeeds. Kindly consider adding validation to the form to prevent that 😊🙏🏻

**Review by MahmoudBakr23 — APPROVED** [review]
Hi team 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by initseis** [comment]
Thanks, @abmAsadullah.

File: scripts/main.js (hunk 1 above)
Comment (initseis) [reply scripts/main.js:59]: Ok, I'll do it.

**Comment by abmAsadullah** [comment]
> Thanks, @abmAsadullah.

Welcome! So our project is accepted!!!

**Pull request merged**